torchrl/envs/libs/vmas.py

```diff
@@ -368,7 +378,7 @@ def _step(
         if not self.het_specs:
             agent_tds = agent_tds.to_tensordict()
         tensordict_out = TensorDict(
-            source={"agents": agent_tds, "done": dones},
+            source={"agents": agent_tds, "done": dones, "terminated": dones},
```

Comment:
so not cloning? what if done is modified in place?

Reply:
Rule of thumb is not to consider in-place modif as a possibility because it's discouraged in pytorch.

In-place modifications in pytorch should never be the rule, and we abide by this rule for efficiency. 

IMO if someone does
```
tensordict.get("done").fill_(0)
```
they are looking for problems.

But we can clone if you think it's safer